frontend/src/bbkit/BBButton.vue

```diff
@@ -0,0 +1,95 @@
+<template>
```

Comment:
@xiaoluoboding PTAL at BBButton

Reply:
`BBButton` with a Tooltip? `Button` is a common name, so we should keep its logic clean, we might call it `BBTooltipButton`?

Reply:
Sounds reasonable.

Reply:
updated, PTAL